test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -863,19 +863,38 @@ def forward(self, x):
         x = torch.randn(2, 3)
         self.run_test(ArithmeticModule(), x)
 
+    def test_floor(self):
+        class FloorModule(torch.nn.Module):
+            def forward(self, x):
+                return torch.floor(x)
+
+        x = torch.tensor([-0.5, 0.0, 0.5, 2], dtype=torch.float32)
+        self.run_test(FloorModule(), x)
+
+    # NOTE: floor division tests currently restrict themselves to only
+    #   test positive division. PyTorch's floor division was previously
+    #   implemented incorrectly and would truncate, not floor, the result
+    #   of the division. For example, -5 // 3 = -2, but, historically,
+    #   torch.floor_divide(-5, 3) = -1.
+    # This behavior has been fixed in PyTorch but the update has not been
+    #   applied to ONNX export yet. See https://github.com/pytorch/pytorch/issues/44692.
```

Comment:
Overall it looks good. 
Maybe updated the comment here with latest fixes. Thanks.